last_practice5_2.java

```diff
@@ -0,0 +1,63 @@
+import java.util.Scanner;
+
+public class last_practice5_2 {
+
+	public static boolean isPrime(int number) {
+
+		// All negative numbers, 0 and 1
+		// are not a prime number
+		if (number <= 1)
+			return false;
+
+		// check for remaining
+		for (int i = 2; i < number; i++)

```

Comment:
you don't need to go over square root of the number. If you haven't find a divisor smaller than square root of the number then you won't find one because sqrt(x) * sqrt(x) = x